packages/browser/core/src/core/auth/internal/StitchUserFactoryImpl.ts

```diff
@@ -19,7 +19,7 @@ import {
   StitchUserProfileImpl
 } from "mongodb-stitch-core-sdk";
 import StitchUser from "../StitchUser";
-import StitchAuthImpl from "./StitchAuthImpl";
+import { StitchAuthImpl } from "./StitchAuthImpl";
```

Comment:
I think you'll be able to make this a default export again.

Reply:
👍